index.html

```diff
@@ -1,33 +1,45 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
```

Comment:
Make sure you add `<meta name="viewport" content="width=device-width, initial-scale=1">` here, otherwise your media queries won't work properly.